org.darktable.Darktable.json

```diff
@@ -78,6 +83,24 @@
                 }
             ]
         },
+        {
+            "name": "lensfun-db",
```

Comment:
You don't need to add a new module for this. Just add another source to the `lensfun` module with the `dest` property set.

Reply:
I just thought of this... could we pull the master branch in a separate module, then just copy of the XML files like I'm doing here? Then the build becomes reproducible, and we can periodically bump the shasum of the master branch.

Reply:
Also in this case: you can add it as another (git) source to the existing module!

Reply:
I think that'd be a better solution, no?

Reply:
@eszlari I'm not sure what the manifest would look like for copying the files around if I add another source to the lensfun module. Can you point me in the right direction?

Reply:
```
post-install:
  cp some_dir /path/...

[...]

type: archive
dest: some_dir
```